sdk/python/foundation-models/system/inference/text-to-image/text-to-image-online-endpoint.ipynb

```diff
@@ -342,7 +342,7 @@
  ],
  "metadata": {
   "language_info": {
-   "name": "python"
+   "name": "ipython"
```

Comment:
Is the language info update intentional?

Reply:
yes.